lib/inv_ca_gcr.cpp

```diff
@@ -283,38 +281,39 @@ namespace quda {
       // note that the heavy quark residual will by definition only be checked every nKrylov steps
       if (total_iter>=param.maxiter || (r2 < stop && !l2_converge) || sqrt(r2/r2_old) < param.delta) {
 
-	if ( (r2 < stop || total_iter>=param.maxiter) && param.sloppy_converge) break;
-	mat(r, x, tmp);
-	r2 = blas::xmyNorm(b, r);  
-	if (use_heavy_quark_res) heavy_quark_res = sqrt(blas::HeavyQuarkResidualNorm(x, r).z);
+        if ( (r2 < stop || total_iter>=param.maxiter) && param.sloppy_converge) break;
+        mat(r, x, tmp);
+        r2 = blas::xmyNorm(b, r);  
+        if (use_heavy_quark_res) heavy_quark_res = sqrt(blas::HeavyQuarkResidualNorm(x, r).z);
 
         // break-out check if we have reached the limit of the precision
-	if (r2 > r2_old) {
-	  resIncrease++;
-	  resIncreaseTotal++;
-	  warningQuda("CA-GCR: new reliable residual norm %e is greater than previous reliable residual norm %e (total #inc %i)",
-		      sqrt(r2), sqrt(r2_old), resIncreaseTotal);
-	  if (resIncrease > maxResIncrease or resIncreaseTotal > maxResIncreaseTotal) {
-	    warningQuda("CA-GCR: solver exiting due to too many true residual norm increases");
-	    break;
-	  }
-	} else {
-	  resIncrease = 0;
-	}
-
-	if ( !convergence(r2, heavy_quark_res, stop, param.tol_hq) ) {
-	  restart++; // restarting if residual is still too great
-
-	  PrintStats("CA-GCR (restart)", restart, r2, b2, heavy_quark_res);
-          blas::copy(*p[0], r);
-
-	  r2_old = r2;
-
-	  // prevent ending the Krylov space prematurely if other convergence criteria not met 
-	  if (r2 < stop) l2_converge = true; 
-	}
-
-	r2_old = r2;
+        if (r2 > r2_old) {
+          resIncrease++;
+          resIncreaseTotal++;
+          warningQuda("CA-GCR: new reliable residual norm %e is greater than previous reliable residual norm %e (total #inc %i)",
+          sqrt(r2), sqrt(r2_old), resIncreaseTotal);
```

Comment:
clang-format suggested the following change:


```
-          warningQuda("CA-GCR: new reliable residual norm %e is greater than previous reliable residual norm %e (total #inc %i)",
-          sqrt(r2), sqrt(r2_old), resIncreaseTotal);
+          warningQuda("CA-GCR: new reliable residual norm %e is greater than previous reliable residual norm %e (total "
+                      "#inc %i)",
+              sqrt(r2), sqrt(r2_old), resIncreaseTotal);
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_ca_gcr.cpp -->